test/scenarios-periodics/el92-src@rpm-standard1.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart-liveimg.ks.template ""
+    launch_vm host1 "rhel-9.2-microshift-source"
+    configure_vm_firewall host1
```

Comment:
Why do we need this here? What's different from other scenarios?

Reply:
The RPM-based system doesn't have the firewall configured by the blueprint. This is the same function that's used in the RPM installation test scenario.

Reply:
We have other scenarios i.e. el92-src@fips.sh using liveimg, but they do not configure firewall explicitly. 
What is different in the current test?